src/core/store.js

```diff
@@ -3,26 +3,40 @@ import config from 'config';
 import { applyMiddleware, compose } from 'redux';
 import { createLogger } from 'redux-logger';
 
+import log from 'core/logger';
+
+export const minimalReduxLogger = () => (next) => (action) => {
```

Comment:
The changes seem fine to me, but I don't really understand everything that is going on. What's actually happening here with this `next` and `action`?

Reply:
The signature of Redux middleware is to call `next(action)` which returns a new state after all reducers have processed the action. Docs: https://redux.js.org/advanced/middleware#the-final-approach